dvtm.c

```diff
@@ -1235,7 +1235,7 @@ focusprevnm(const char *args[]) {
 
 static void
 focuslast(const char *args[]) {
-	if (lastsel)
+	if (lastsel && isvisible(lastsel))
```

Comment:
I am not very familiar with this code, but shouldn't we make lastsel visible instead?